Courses/_lib/src/course.js

```diff
@@ -327,7 +327,7 @@ $(document).ready(function () {
   if (CodeMirror.colorize) {
     CodeMirror.colorize(document.body.getElementsByTagName("pre"), 'cypher');
   }
-  Intercom('trackEvent', window.intercomEvent + '-part' + trainingPartIndex)
+  Intercom('trackEvent', 'PAGE_VIEW', {course: window.trainingClassName, module: trainingPartIndex })
```

Comment:
So we are sending an event when the page is loaded and then we are sending the same event when the user is authenticated (and after calling `Intercom('update', {})`, see above)

Reply:
I think we might only need to track the page view event. I am guessing intercom should be able to retrospectively associate the event after the user is set using `Intercom('update')`. I have asked the intercom team to confirm on that. If so, then we only need to call it here and can remove the one after login.

Reply:
Ok :+1: